libraries/SRV_Channel/SRV_Channel.h

```diff
@@ -147,6 +147,7 @@ class SRV_Channel {
         k_LED_neopixel2         = 121,
         k_LED_neopixel3         = 122,
         k_LED_neopixel4         = 123,
+        K_wingsail_elevator     = 124,
```

Comment:
little typo on the case of the "K".  should be a small "k"

Reply:
fixed